packages/interactor/src/definitions/html.ts

```diff
@@ -19,6 +19,16 @@ const HTMLInteractor = createInteractor<HTMLElement>('element')
     blur: ({ perform }) => perform((element) => { element.blur(); }),
   })
 
+// Because JSDOM does not have any concept of flow or layout, it cannot actually implement
+// `innerText` correctly, so they have opted not to implement it at all. So, to make things
+// work use `textContent` as a backup for all `HTMLElement` subtypes if `innerText` is
+// undefined
+//
+// See details: https://github.com/jsdom/jsdom/issues/1245
+export function innerText(element?: HTMLElement): string {
+  return (element?.innerText != null ? element?.innerText :  element?.textContent) || '';
```

Comment:
I suppose this works because `innerText` will always return a string, right? So if it is `undefined` that means it actually isn't implemented?

Reply:
Exactly!